examples/standard/variables.tf

```diff
@@ -0,0 +1,193 @@
+  type        = bool
+  default     = false
+}
+
+variable "configure_proxy" {
+  description = "Specify if SQUID proxy will be configured. If yes, ensure 'proxy_config' optional variable is set properly. Proxy is mandatory for the landscape, so set this to 'false' only if proxy already exists."
+  type        = bool
+  default     = true
+}
+
+variable "configure_ntp_forwarder" {
+  description = "Specify if DNS forwarder will be configured. If yes, ensure 'dns_config' optional variable is set properly."
+  type        = bool
+  default     = true
+}
+
+variable "configure_nfs_server" {
+  description = "Specify if DNS forwarder will be configured. If yes, ensure 'dns_config' optional variable is set properly."
+  type        = bool
+  default     = true
+}
+
+variable "configure_dns_forwarder" {
+  description = "Specify if DNS forwarder will be configured. If yes, ensure 'dns_config' optional variable is set properly."
+  type        = bool
+  default     = false
+}
+
+variable "access_host_or_ip" {
+  description = "Jump/Access server public host name or IP address. This host name/IP is used to reach the landscape."
+  type        = string
+}
+
+variable "internet_services_host_or_ip" {
+  description = "Host name or IP address of the virtual server instance where proxy server to public internet and to IBM Cloud services will be configured."
+  type        = string
+}
+
+variable "private_services_host_or_ip" {
+  description = "Default private host name or IP address of the virtual server instance where private services should be configured (DNS forwarder, NTP forwarder, NFS server). Might be empty when no services will be installed. Might be overwritten in the optional service specific configurations (in order to install services on different hosts)."
+  type        = string
+  default     = null
+}
+
+#####################################################
+# Optional Parameters
+#####################################################
+
+variable "pvs_management_network" {
+  description = "PowerVS Management Subnet name and cidr which will be created."
+  type        = map(any)
+  default = {
+    "name" = "mgmt_net"
+    "cidr" = "10.51.0.0/24"
+  }
+}
+
+variable "pvs_backup_network" {
+  description = "PowerVS Backup Network name and cidr which will be created."
+  type        = map(any)
+  default = {
+    "name" = "bkp_net"
+    "cidr" = "10.52.0.0/24"
+  }
+}
+
+variable "tags" {
+  description = "List of Tag names for PowerVS service"
+  type        = list(string)
+  default     = null
+}
+
+variable "transit_gateway_name" {
+  description = "Name of the existing transit gateway. Required when creating new cloud connections"
+  type        = string
+  default     = null
+}
+
+variable "cloud_connection_speed" {
+  description = "Speed in megabits per sec. Supported values are 50, 100, 200, 500, 1000, 2000, 5000, 10000. Required when creating new connection"
```

Comment:
Could you add a validation block to check inputs

Reply:
Sure will include it

Reply:
Still not addressed

Reply:
I didnt handle it because the apply itself throws an error saying not  right values, But now i realized that, it can be caught during plan itself

Reply:
NOt required, In plan the provider itself throws an error if not valid